### ISSUE TITLE
xorgxrdp(latest) build failure on Ubuntu 20.04

### ISSUE THREAD

**Issue by i928** [neutrinolabs/xrdp#1868]
rdpClientCon.c: In function ‘rdpClientConProcessMsgClientInfo’:
rdpClientCon.c:845:31: error: ‘struct xrdp_client_info’ has no member named ‘version’
  845 |     if (clientCon->client_info.version != CLIENT_INFO_CURRENT_VERSION)
      |                               ^
rdpClientCon.c:845:43: error: ‘CLIENT_INFO_CURRENT_VERSION’ undeclared (first use in this function)
  845 |     if (clientCon->client_info.version != CLIENT_INFO_CURRENT_VERSION)
      |                                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~
rdpClientCon.c:845:43: note: each undeclared identifier is reported only once for each function it appears in
rdpClientCon.c:849:42: error: ‘struct xrdp_client_info’ has no member named ‘version’
  849 |                    clientCon->client_info.version));
      |                                          ^
rdpClientCon.c:56:43: note: in definition of macro ‘LLOGLN’
   56 |     do { if (_level < LOG_LEVEL) { ErrorF _args ; ErrorF("\n"); } } while (0)
      |                                           ^~~~~
make[3]: *** [Makefile:678: rdpClientCon.lo] Error 1
make[3]: Leaving directory '/home/chiiwen/xorgxrdp/module'
make[2]: *** [Makefile:697: all-recursive] Error 1
make[2]: Leaving directory '/home/chiiwen/xorgxrdp/module'
make[1]: *** [Makefile:434: all-recursive] Error 1
make[1]: Leaving directory '/home/chiiwen/xorgxrdp'
make: *** [Makefile:366: all] Error 2

built ok with 0.9.15

**Comment by metalefty** [comment]
Duplicate of neutrinolabs/xorgxrdp#189.

**Comment by metalefty** [comment]
BTW, you should report xorgxrdp issue to xorgxrdp repo. Then you’ll find the duplicate issue before you raise a new issue :)
